Handles MeetingID with spaces (issue #40)

glunardi opened the pull request:
The BBB_MEETING_ID environment variable does not get passed around correctly if there is a space in the MeetingID.

Proposing to wrap BBB_MEETING_ID in double-quotes to enable support for MeetingID with spaces and other extended characters probably.

glunardi committed:
Handles MeetingID with spaces (issue #40)

The BBB_MEETING_ID environment variable does not get passed around correctly if there is a space in the MeetingID.

Proposing to wrap BBB_MEETING_ID in double-quotes to enable support for MeetingID with spaces and other extended characters probably.

mtsonline reviewed (changes requested):
Hi and thanks for the pullrequest. Would you mind to add the quoting for all arguments - so title and so on will allow spaces too? cheers

glunardi committed:
Wrapping {BBB_USER_NAME} in quotes (issue #40)

Looks like {BBB_USER_NAME} is another string variable that could break without quotes. The other variables should be fine and $MEETING_TITLE was already wrapped.

glunardi committed:
Wrapping passwords just in case (issue #40)

glunardi commented:
It looks like "{BBB_CHAT_NAME}" would be the only other string that would need to be wrapped (we intend to use a space in our BBB chat name :-)".

Sorry, have not slept in a while. in the non-chat capture I wrapped BBB_USER_NAME too but now that I think about it, it is quite possible a user name would have extended characters for some users. And while at it... will include the passwords

mtsonline reviewed (approved):
thanks for the changes - feel free to add other improvements ...
